core/runtime/src/main/java/io/quarkiverse/langchain4j/runtime/aiservice/AiServiceMethodImplementationSupport.java

```diff
@@ -428,6 +430,7 @@ private static Optional<SystemMessage> prepareSystemMessage(AiServiceMethodCreat
         }
 
         templateParams.put(ResponseSchemaUtil.templateParam(), createInfo.getResponseSchemaInfo().outputFormatInstructions());
+        templateParams.put("chat_memory", previousChatMessages);
```

Comment:
Not related to this PR, but it would be great to have a list of all the variables we handle (like current_date, response_schema...)

Reply:
Yeah, we should improve the docs to include all these